Sources/MapboxMaps/Gestures/GestureManager.swift

```diff
@@ -64,6 +64,11 @@ public final class GestureManager: GestureHandlerDelegate {
         return quickZoomGestureHandler.gestureRecognizer
     }
 
+    /// The gesture recognizer for the single tap gesture
```

Comment:
Would like to see a description of how this relates to annotation interaction delegate

Reply:
Hmm not sure if that context should live in the gesture manager. Seems like some information that should be housed in the annotation manager.

Reply:
why not both?

I'm thinking about the scenario we discussed yesterday about how it might be unclear to developers what the effect of disabling this gesture recognizer might be.